Blockzilla/Utilities/URIFixup.swift

```diff
@@ -7,7 +7,7 @@ import Foundation
 class URIFixup {
     static func getURL(entry: String) -> URL? {
         let trimmed = entry.trimmingCharacters(in: CharacterSet.whitespacesAndNewlines)
-        guard let escaped = trimmed.addingPercentEncoding(withAllowedCharacters: CharacterSet.urlAllowed) else {
+        guard let escaped = trimmed.addingPercentEncoding(withAllowedCharacters: CharacterSet.alphanumerics) else {
```

Comment:
I'm not sure if this is the right way to go. :(

Reply:
Not sure either, there other `CharacterSet` variables like `urlQueryAllowed` `urlHostAllowed` and I guess `urlAllowed` was created to combine all of them. Is there a reason we want to change that?

Reply:
The changes around this are in order to make this PR working on Xcode 15 and have Focus on that xcode / OS version.
That said, I'm not sure about those changes either...
These were the changes done on Firefox to be able to upgrade to that newer version:
https://github.com/mozilla-mobile/firefox-ios/commit/21b865bd7b39845c3202702428b039a456082ea3#diff-8e531330b32bc0064301e0740a902fcfd7b0c9069e3f737fee94dd936417e28fL9
We may need to do something similar

Reply:
I looked at that diff, too, and tried to add such an `init()` to `extention URL {}` class. The test in question is only on `URLFixup` and those `extenion URL` is not related.